Solutions/Authomize/Analytic Rules/Admin_SaaS_account_detected.yaml

```diff
@@ -0,0 +1,63 @@
+id: 87419138-d75f-450d-aca4-1dc802e32540
+name: Admin SaaS account detected
+kind: Scheduled
+description: The rule detects internal admins accounts, it's recommended to review any new administrative permission.
+severity: High
```

Comment:
This rule does not seems to be a High. It can be Low as the description reads recommended.

Reply:
Updated